examples/docker-bookinfo.yaml

```diff
@@ -56,28 +58,19 @@ ratings-v1:
 reviews-v1:
   image: amalgam8/a8-examples-bookinfo-reviews-sidecar:v1
   environment:
-    - A8_REGISTRY_URL=http://registry:8080
     - A8_SERVICE=reviews:v1
-    - A8_REGISTER=true
-    - A8_ENDPOINT_PORT=9080
-    - A8_HEALTHCHECKS=http://localhost:9080/health
   external_links:
     - registry
 
 reviews-v2:
   image: amalgam8/a8-examples-bookinfo-reviews-sidecar:v2
   environment:
-    - A8_LOG=true
-    - A8_REGISTER=true
     - A8_PROXY=true
-    - A8_REGISTRY_URL=http://registry:8080
     - A8_SERVICE=reviews:v2
-    - A8_ENDPOINT_PORT=9080
     - A8_LOGSTASH_SERVER='logstash:8092'
     - A8_CONTROLLER_URL=http://controller:8080
     - A8_CONTROLLER_POLL=5s
     - A8_REGISTRY_POLL=5s
```

Comment:
Can we move all this into config.yaml for reviews service? May be we can just have two configs. config-reviews-v1.yaml and config-reviews-v2v3.yaml.

This makes the deployment script look much more simple.